### DISCUSSION TITLE
2024.11.28 wiredV2模型提升对轻度旋转表格识别准确率，参见输入参数。 这是模型更新了？还是只是处理代码更新了？

### DISCUSSION THREAD

**Discussion by nissansz** [RapidAI/TableStructureRec#92] [Q&A]
2024.11.28 wiredV2模型提升对轻度旋转表格识别准确率，参见输入参数。 这是模型更新了？还是只是处理代码更新了？

**Comment by Joker1212** [comment]
只是改了后续处理代码，模型没有再训练

**Comment by nissansz** [comment]
单字ocr匹配
# 将单字box转换为行识别同样的结构)
from rapidocr_onnxruntime import RapidOCR
from wired_table_rec.utils_table_recover import trans_char_ocr_res
img_path = "tests/test_files/wired/table4.jpg"
ocr_engine =RapidOCR()
ocr_res, _ = ocr_engine(img_path, return_word_box=True)
ocr_res = trans_char_ocr_res(ocr_res)


单字ocr是单独模型，还是计算的估计单字坐标？

**Reply by Joker1212** [reply]
ctc loss后置处理计算的坐标

**Comment by nissansz** [comment]
有什么好的单字模型吗？paddle那个配置是训练单字的？

**Comment by nissansz** [comment]
文本行检测模型，有modelscope的模型可以用到rapid里面吗？

**Comment by nissansz** [comment]
https://github.com/MGzhou/duguang-ocr-onnx
https://huggingface.co/Luoaho/duguang-ocr-onnx

这里好像有读光的检测和识别模型，不知道在rapid里面怎么使用

**Reply by Joker1212** [reply]
两个模型的输出和前置后置处理逻辑不同，应该不能直接使用，rapidOCR暂时没有集成读光的模型，原仓库作者其实已经简化操作了，可以直接使用

**Comment by nissansz** [comment]
试用了seglink++， 准确率挺高的。
不知道怎么自己训练读光
